source/hic_hal/stm32/stm32h743ii/extended_features/source/udb_power_measurement.c

```diff
@@ -0,0 +1,89 @@
+#include "udb_power_measurement.h"
+#include "pac193x.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+static pac193x_cfg_t s_udb_power_measurement_cfg;
+
+static const float k_udb_power_measurement_resister[] =
```

Comment:
Spelling is "resistor". I'm not sure if "resister" is an actual word, but if it is, then it would mean something more like an individual person who is actively resisting something, whereas "resistor" somehow feels more like an object that provides resistance (or maybe I have that association because of my experience with electronics). Unfortunately, I can't find any clear-cut rule for this difference in spelling, I think it's more of a case-by-case basis: https://english.stackexchange.com/a/4734

Reply:
Yes, I just misspelled the word resistor.